profile/README.md

```diff
@@ -0,0 +1,23 @@
+## Hey, it's 👋 Amplication here!
+
+![Amplication](https://pbs.twimg.com/profile_banners/1294979453005770753/1656831372/1500x500)
```

Comment:
please move this image to our repo

Reply:
can you provide a high resolution img. of the banner?

Reply:
![cover v2](https://user-images.githubusercontent.com/53312820/179914535-6617eec2-e305-4a8b-8063-db3bdf0f427c.png)

Here you go @AnujSsStw